src/main.rs

```diff
@@ -83,8 +85,9 @@ fn increase_counter(counter: &mut Vec<u8>, start: Option<usize>) {
 	loop {
 		if counter[i] < 255 {
 			counter[i] += 1;
-			break
-		} else if counter[i] == 255 {
+			break;
+		} else {
+			// else if counter[i] == 255 unnecessary
```

Comment:
Nice change, but please remove the comment before merging

Reply:
Done!